routes/web.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/**
+ * Because these routes should mimic static files, no middlewares are applied.
+ */
+
+use Illuminate\Support\Facades\Route;
+use JustBetter\GlideDirective\Controllers\ImageController;
+
+$patterns = [
+    'file' => '.*',
+    'format' => '\..+',
+];
+
+Route::get('glide-image/placeholder/{file}', [ImageController::class, 'placeholder'])
+    ->where($patterns)
+    ->name('glide-image.placeholder');
+
+Route::get('img/'.config('justbetter.glide-directive.storage_prefix', 'storage/glide-image').'/{preset}/{fit}/{s}/{file}{format}', [ImageController::class, 'getImageByPreset'])
```

Comment:
Do we need to place it in `/img/` we could either place them in `img` and leave out the `storage` prefix, or place them in `storage` and leave out the `img` prefix.

By the looks of it, this route currently does not match up with the place where we save the file https://github.com/justbetter/statamic-glide-directive/pull/20/files#diff-0f11c6b25af48e40fd2d6ca4eeaf1b4313ca9e92db330ddfec459424a95485f3R93

Reply:
Changed this around a bit, `/img` is de default cache folder for Statamic. I made a new config for this in case this has been changed.

Also removed `storage` out of the prefix, as its not needed. The new structure would now be `/img/glide-image/....`